core/src/ast/expr.rs

```diff
@@ -290,15 +283,6 @@ mod tests {
             .to_sql()
         );
 
-        assert_eq!(
```

Comment:
This is a unit test that will be added later for the to_sql() implementation for the enum Function.
Therefore, it will disappear temporarily.